contracts/integrations/test/utils/deployment_manager.ts

```diff
@@ -366,7 +361,17 @@ export class DeploymentManager {
             stakingArtifacts,
             stakingLogic.address,
         );
-        const stakingWrapper = new TestStakingContract(stakingProxy.address, environment.provider);
+
+        const logDecoderDependencies = _.mapValues(
```

Comment:
This has gone back and forth a few times now. Why do we need the `logDecoderDependencies`?

Reply:
m yeah I think we don't need it

Reply:
Ok cool. Just checking. I've seen it pop up in the staking wrapper, so I actually think that I've put it in here before